tools/build_pytorch_libs.sh

```diff
@@ -217,19 +212,13 @@ function build_nccl() {
               -DCMAKE_CXX_FLAGS="$C_FLAGS $CPP_FLAGS $USER_CFLAGS" \
               -DCMAKE_SHARED_LINKER_FLAGS="$USER_LDFLAGS" \
```

Comment:
I believe you need to pass `NUM_JOBS` to `cmake` for `third_party/nccl/CMakeList.txt` to use it., e.g., `-DNUM_JOBS="$NUM_JOBS" \`